packages/components/src/molecules/Rating/Rating.tsx

```diff
@@ -47,6 +47,7 @@ const Rating = forwardRef<HTMLUListElement, RatingProps>(function Rating(
   ref
 ) {
   const [hover, setHover] = useState(0)
+  let ratingIcon = icon ? icon : <Star />
```

Comment:
```suggestion
  const ratingIcon = icon ? icon : <Star />
```

Reply:
Actually, `<Star />` could be the default value for the `icon` prop, what do you think?

Reply:
hmm yeah, but we should allow the user to user other icons if needed.